pricebots/ethmaxy/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "galleon-ethmaxy-pricebots",
```

Comment:
@ADWilkinson Can you check the naming on the packages, not sure which format you would like to use now that the bots are split